apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '../../util/color';
+import styleConstants from '../../styleConstants';
+import Button from '../Button';
+import {navigateToHref} from '@cdo/apps/utils';
+
+const BorderedCallToAction = props => {
+  const {
+    headingText,
+    descriptionText,
+    className,
+    buttonText,
+    buttonUrl,
+    buttonClass,
+    onClick,
+    solidBorder
+  } = props;
+
+  const borderStyle = solidBorder ? styles.solidBorder : styles.dashedBorder;
+
+  return (
+    <div style={[styles.outerBox, borderStyle]} className={className}>
+      <div style={styles.wordBox}>
+        <div style={styles.heading}>{headingText}</div>
+        <div style={styles.description}>{descriptionText}</div>
+      </div>
+      <Button
+        onClick={onClick || (() => navigateToHref(buttonUrl))}
```

Comment:
this was probably pre-existing behavior, but what happens (or should happen) if `onClick` and `buttonUrl` are both empty?

Reply:
Great question ... it won't be pretty. I've decided to throw an error if that happens. Open to other ideas too!